_posts/2020-03-29-newpipe-0.19.0-released.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "NewPipe 0.19.0 released: Folder your feeds and mute them"
+short: "NewPipe 0.19.0 released"
+date: 2020-03-29 00:00:00 +0100
+categories: [release, pinned]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+A new day, a new full release. In today's update, NewPipe is proudly bringing you a great way to sort your subscriptions and a mute button, on top of the usual smaller improvements and fixes<!-- more -->
+
+
+## Folders and fixed Feeds
```

Comment:
We should be consistent here and call this feature "Feed Groups" as it is called in the app.

Reply:
Any thoughts on this? Do you agree / disagree?

Reply:
"WHAT'S NEW FEED IS FIXED" should definitely be highlighted very much. "Feed order and subscription groups" is the name of the PR, and I think that is a suitable title.

Actually, even though they are part of one PR, they should be separate headings. Maybe:

## What's New now shows the latest videos first! (Finally!)

and

## Subscription Groups

Reply:
> What's New now shows the latest videos first! (Finally!)

if I put this in its own heading, it wouldnt get much content. Rn its like two sentences

Reply:
It's a major pet peeve of users (going by issue comments on Github, as well as personal feedback from friends). It should be highlighted strongly in some way, if not its own heading.

Reply:
okay, it has its own heading "now"